docs/02-architecture/diagrams/descriptions/architecture/52a-chembl-pipeline-activity-gold-fields-1.md

```
@@ -0,0 +1,39 @@
+______________________________________________________________________
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)